widgets/common/src/WorkflowsMenu.jsx

```diff
@@ -4,6 +4,11 @@
 
 const WorkflowsPropType = Stage.PropTypes.Workflows;
 
+function filterWorkflows(workflows) {
```

Comment:
Minor:

How about 

```suggestion
function workflowsWithoutUpdate(workflows) {
```

for providing a bit more context :smile: ? I'm not 100% sure this makes sense

Reply:
Could be, but if we are going to be pure then shouldn't it be:
```suggestion
function getWorkflowsWithoutUpdate(workflows) {
```
?

Reply:
Yeah, that would make sense :smile: But I see it's too late :slightly_smiling_face: